src/main/java/fi/metatavu/metamind/bot/BotController.java

```diff
@@ -110,35 +111,50 @@ public MetamindBot getBotInstance(Story story) {
     String storyJson = story.getStoryJson();
     
     Map<String, SlotModel> slotModels = new HashMap<>();
-    IntentModel intentModel = null;
+    List<IntentModel> intentModels = new ArrayList<IntentModel>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)